spyder/plugins/editor/panels/linenumber.py

```diff
@@ -97,6 +98,7 @@ def draw_pixmap(xleft, ytop, pixmap):
 
         for top, line_number, block in self.editor.visible_blocks:
             data = block.userData()
+            self.paint_cell(painter)
```

Comment:
Same as the other call done in the debugger panel (probably we should move this outside the `for` statement or make the method work passing a specific `block` as argument)